src/components/portfolio/FormUploadPortfolio.jsx

```diff
@@ -0,0 +1,23 @@
+const FormUploadPortfolio = ({onSubmit, onChange, multiple}) => {
```

Comment:
Valya!

That’s great! I can add a welcome video and files to my portfolio.

However, when I’ve already uploaded a file and then try to add more but forget to select a new file, and push upload, the request still goes through and reuses the previously uploaded file(s).

Reply:
fixed